src/validation.cpp

```diff
@@ -2233,7 +2233,17 @@ static int64_t nTimePostConnect = 0;
  * part of a single ActivateBestChainStep call.
  */
 struct ConnectTrace {
+private:
     std::vector<std::pair<CBlockIndex*, std::shared_ptr<const CBlock> > > blocksConnected;
+
+public:
+    void BlockConnected(CBlockIndex* pindex, const std::shared_ptr<const CBlock>& pblock) {
```

Comment:
I prefer to force peoplt to not take the atomic increment hit without changing the code :).